test/functional/page_objects/header_page.js

```diff
@@ -166,7 +166,9 @@ export function HeaderPageProvider({ getService, getPageObjects }) {
       log.debug('clickGoButton');
       await retry.try(async () => {
         remote.setFindTimeout(defaultFindTimeout);
-        await testSubjects.click('timepickerGoButton');
+        // Sometimes EUI toast notification covers the go button
+        const elem = await testSubjects.find('timepickerGoButton');
+        await remote.execute('arguments[0].click();', [elem]);
```

Comment:
If this way to click elements is always better than what testSubjects.click is using, then we should adjust it there so everything is using this kind of click.

I have seen some issues when trying to click of getting an error like "this element is blocked by something else" when it's not a toast and there is nothing on the screen actually blocking the element. I never figured out what the issue was, and just slightly modified code to work around it, but maybe this would have fixed it. 🤷‍♀️   But I wouldn't want to see this kind of click as just a one-off here.